### PR TITLE
contributers list

### PR DESCRIPTION
**Pull request by Kobe-Y**
My branches are acting weird, i did not mean to create a new master. my b

### CODE REVIEW

File: Contributors.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      <div className="w-1/3 bg-neutral-300 flex flex-col justify-center items-center">
+        <div className="text-center">
+          <p className="text-blue-500 font-bold font-serif text-2xl md:text-3xl lg:text-4xl leading-8 m-0 mt-10 mr-20">
+            Contributers: <br />
+          </p>
+          {/* Paragraph with added padding */}
+          <ul>
+            <li>Cody Lin</li>
+            <li>Kobe Ying</li>
+            <li>Arnav Swami</li>
```
Comment (JaviChip) [inline Contributors.jsx:18]: Can you make these the same text size as Mission Statement and Notes

File: Contributors.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+const Contributers = () => {
+  return (
+    <div className="bg-neutral-300 flex w-full justify-center items-center">
+      {/* Image container with 1/3 width and added padding */}
+
+      {/* Content container with 2/3 width */}
+      <div className="w-1/3 bg-neutral-300 flex flex-col justify-center items-center">
+        <div className="text-center">
+          <p className="text-blue-500 font-bold font-serif text-2xl md:text-3xl lg:text-4xl leading-8 m-0 mt-10 mr-20">
```
Comment (JaviChip) [inline Contributors.jsx]: can you use text-blue-800 for this

Comment (Kobe-Y) [reply Contributors.jsx]: Do you want the list to be blue-800 or just the "Contributors: " part?

Comment (JaviChip) [reply Contributors.jsx]: Just Contributors Text

**Review by JaviChip — APPROVED** [review]

**Pull request merged**